core/ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/TextBox.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.ui;
+
+/**
+ * An Extension of GWT TextBox.
+ * Contains methods for marking the text box containing valid or invalid values.
+ *
+ * @author Vitaliy Guliy
+ */
+public class TextBox extends com.google.gwt.user.client.ui.TextBox {
```

Comment:
I think `ValidationTextBox` or smth like that, name would be more clear to user instead of simple `TextBox`. Wdyt?